compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUVectorToGPU.cpp

```diff
@@ -137,19 +137,12 @@ struct LLVMGPUVectorToGPUPass
                                             std::move(flatternpatterns)))) {
       return signalPassFailure();
     }
-    RewritePatternSet patterns(funcOp.getContext());
-    mlir::vector::populateCastAwayVectorLeadingOneDimPatterns(patterns);
-    populatePrepareVectorToMMAPatterns(patterns, llvmgpuUseMMASync);
-    if (failed(applyPatternsAndFoldGreedily(getOperation(),
-                                            std::move(patterns)))) {
-      return signalPassFailure();
-    }
```

Comment:
any reason to remove those patterns here? Sounds like we apply `populateCastAwayVectorLeadingOneDimPatterns` later. I would leave it as is